opencv_warco/package.xml

```diff
@@ -10,7 +10,7 @@
   <!-- Commonly used license strings: -->
   <license>MIT</license>
 
-  <url type="website">https://github.com/lucasb-eyer/opencv-warco</url>
+  <url type="website">https://github.com/lucasb-eyer/opencv_warco</url>
```

Comment:
Also the executable targets shouldn't have been renamed... It's getting late; I will attribute it to that. I hope that doesn't cause any problems.